src/ResourceCollection.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace silverorange;
+
+use Countable;
+use Serializable;
+use IteratorAggregate;
```

Comment:
for these simple built-in interfaces, just use `\Countable` rather than aliasing them explicitly.